locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -303,6 +306,56 @@ situations because **it allows you to "starve" your I/O by making
 recursive `process.nextTick()` calls**, which prevents the event loop
 from reaching the **poll** phase.
 
+#### Deduplication
```

Comment:
I am not sure if this is the correct place. The next section answers why the behaviour mentioned in the line above is allowed.

Reply:
Hmm, you're right. Let me fix this.